.changeset/eighty-falcons-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/sitemap": patch
```

Comment:
As we are adding a new option, this should be a `minor` and not a `patch`

Reply:
I changed it.